### PR TITLE
docs(react-scheduler): add description to AppointmentForm Select's availableOptions

### PR DESCRIPTION
**Pull request by ndom91**
Updated `AppointmentForm` `selectProps` documentation to detail the required shape of the `availableOptions` object. 

I was looking around for forever for this and had to dig into the code to find it eventually. 

Go open-source :wink: :thumbsup:

### CODE REVIEW

**Comment by AryamnovEugeniy** [comment]
Hello @ndom91 !

Thank you for your interest in our product!
We will help you to merge this Pull Request.

File: packages/dx-react-scheduler/docs/reference/appointment-form.md [hunk 1]
```diff
@@ -53,6 +53,15 @@ weeklyRecurrenceSelectorComponent | ComponentType&lt;[AppointmentForm.WeeklyRecu
 
 ## Interfaces
 
+### SelectOption
+
+Represents a Select Editor's value and its id.
```
Comment (RomanTsukanov) [inline packages/dx-react-scheduler/docs/reference/appointment-form.md]:
```suggestion
An option in the Select editor.
```

File: packages/dx-react-scheduler/docs/reference/appointment-form.md [hunk 2]
```diff
@@ -53,6 +53,15 @@ weeklyRecurrenceSelectorComponent | ComponentType&lt;[AppointmentForm.WeeklyRecu
...
+
+Field | Type | Description
+------|------|------------
+id | string &#124; number | The option's id.
+text | string | The option's text value.
```
Comment (RomanTsukanov) [inline packages/dx-react-scheduler/docs/reference/appointment-form.md]:
```suggestion
text | string | The option's text.
```

**Comment by ndom91** [comment]
Thanks guys, the updated display of this information is much better too ;)

**Comment by MaximKudriavtsev** [comment]
@ndom91,

You are welcome!

**Pull request merged**